protoc-gen-swagger/options/annotations.proto

```diff
@@ -35,3 +35,10 @@ extend google.protobuf.ServiceOptions {
   // different descriptor messages.
   Tag openapiv2_tag = 1042;
 }
+extend google.protobuf.FieldOptions {
+  // ID assigned by protobuf-global-extension-registry@google.com for grpc-gateway project.
+  //
+  // All IDs are the same, as assigned. It is okay that they are the same, as they extend
+  // different descriptor messages.
+  JSONSchema openapiv2_field_json_schema = 1042;
```

Comment:
Can we name this just `openapiv2_field` similar to what was done with other options?

Reply:
I've fixed it in https://github.com/grpc-ecosystem/grpc-gateway/pull/687/commits/48e32c98afbc0758fbb4c635e0f67b601532f49c